lib/embulk/input/jira.rb

```diff
@@ -62,6 +62,9 @@ def self.guess(config)
         retry_initial_wait_sec = config.param(:retry_initial_wait_sec, :integer, default: 1)
         retryer = retryer(retry_limit, retry_initial_wait_sec)
 
+        # Get credential before going to search issue
+        jira.get_user_credential(username)
```

Comment:
Method name is `get_xxx` but we don't use that result.
That method does check/validate/authenticate/etc actually, so renaming it would be better.